src/components/10-atoms/text/ui.test.js

```diff
@@ -141,21 +143,45 @@ test('should have correct font definitions for text size 2 with custom span tag'
     .eql('27px');
 });
 
-
 fixture('Text - Bold')
   .page(`${host}/iframe.html?id=atoms-text--text&knob-variant=bold`)
   .beforeEach(async t => {
     await t.resizeWindow(380, 680);
   });
 
 test('should have correct font weight for text bold', async t => {
+  const $axaElemShadow = await Selector(() =>
+    document.querySelector('axa-text[variant="bold"]').querySelector('.a-text')
```

Comment:
I would make a selector with the two selectors combined

Reply:
I did not touch this code, that is git having that part messed up.

Reply:
I would still do it anyway. is a small thing. and i recon is prettier, not git.

Reply:
Aha, I misunderstood.